staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -263,7 +264,13 @@ func (a *mutatingDispatcher) callAttrMutatingHook(ctx context.Context, h *admiss
 		}
 	}
 
-	if err := r.Do(ctx).Into(response); err != nil {
+	wd, ok := endpointsrequest.WebhookDurationFrom(ctx)
```

Comment:
nit: maybe something like:
```
do := func() { err = r.Do(ctx).Into(response)
if wd, ok := endpointsrequest.WebhookDurationFrom(ctx); ok {
  do = func() { wd.AdmitTracker.Track(do())
}
do()
```

Reply:
Unfortunately I think this causes infinite recursion. (https://play.golang.org/p/ug1ROGHRMM0)

Reply:
How about:
```
do := func() { err = r.Do(ctx).Into(response)
if wd, ok := endpointsrequest.WebhookDurationFrom(ctx); ok {
  tmp := do
  do = func() { wd.AdmitTracker.Track(do())
}
do()
```